backend/src/models/teams.rs

```diff
@@ -264,6 +269,23 @@ impl teams::Model {
 
         Ok(())
     }
+
+    pub async fn confirm(
+        database: &DatabaseConnection,
+        confirmation_code: Uuid,
+    ) -> Result<(), Error> {
+        let team = teams::Entity::find()
+            .filter(teams::Column::ConfirmationCode.eq(confirmation_code))
+            .one(database)
+            .await?
+            .ok_or(Error::Team(TeamNotFound))?;
+
+        let mut active_team: ActiveModel = team.into();
```

Comment:
Model should be responsible for logic related to itself. Create a service or run 2 functions in route